plugins/modules/elb_classic_lb_info.py

```diff
@@ -157,22 +157,34 @@
 def list_elbs(connection, load_balancer_names):
     results = []
 
+    if not load_balancer_names:
+        for lb in get_all_lb(connection):
+            results.append(describe_elb(connection, lb))
+
     for load_balancer_name in load_balancer_names:
```

Comment:
Having a quick look at the boto3 docs, it should be possible to call describe_load_balancers with no args:

```
response = client.describe_load_balancers()
```

This is generally better than running get_all_lb and then describe_load_balancers for every LB (1 API call rather than many).

If you're feeling adventurous, then using a paginator would also be a good idea....

Reply:
https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/elb.html#ElasticLoadBalancing.Client.describe_load_balancers

Reply:
It sounds better actually, I will dig to this way.